include/revng/PipelineC/Prototypes.h

```diff
@@ -184,6 +182,19 @@ const char *rp_manager_produce_targets(rp_manager *manager,
                                        rp_step *step,
                                        rp_container *container);
 
+/**
+ * Request to run the required analysis
+ *
+ * \param tagets_count must be equal to the size of targets.
+ *
+ * \return 0 if an error was encountered, the serialized model diff otherwise
+ */
+const char *rp_manager_run_analysis(rp_manager *manager,
+                                    uint64_t targets_count,
+                                    rp_target *targets[],
```

Comment:
So this has to become a list of lists right?
Please put this PR in draft if it's not ready.